tests/fast-integration/eventing-test/eventing-test.js

```diff
@@ -124,6 +128,12 @@ describe('Eventing tests', function() {
         eventingSubscription(`sap.kyma.custom.commerce.order.created.v1`, sink, 'order-created', testNamespace),
       ];
 
+      before('check if at least once delivery tests need to be skipped', async function() {
```

Comment:
(nit)
I would better log, when skip the test and not the check. Something like:

```
 before('', async function() {
        if (skipAtLeastOnceDeliveryTest()) {
          console.log('Skipping the at least once delivery tests for non-limits retention policy');
          this.skip();
        }
      });
```